.golangci.yml

```diff
@@ -130,14 +129,14 @@ issues:
   - linters:
     - dupl
     - scopelint
-    path: "_test\\.go"
+    path: _test\.go # regexp
   - linters:
     - forbidigo
     - gosec
-    path: "internal/"
+    path: internal/ # regexp
   - linters:
-    - goerr113
+    - err113
     text: do not define dynamic errors, use wrapped static errors instead
   - linters:
     - gochecknoinits
-    path: "docs/docs.go"
+    path: docs\.go # regexp
```

Comment:
I changed this because it didn't appear to work as expected inside VS Code for me, despite seemingly working in my terminal. Strange...